<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="ajay" ignored="0" name="Default test" tests="18" failures="2" timestamp="01 Feb 2018 15:57:35 GMT" time="188.209" errors="0">
  <testcase name="openingRechargeApp" time="59.111" classname="test.horoscope.Total_horoscope">
    <failure type="java.lang.AssertionError" message="expected [Recharge successful] but found [Recharge failed]">
      <![CDATA[java.lang.AssertionError: expected [Recharge successful] but found [Recharge failed]
	at org.testng.Assert.fail(Assert.java:93)
	at org.testng.Assert.failNotEquals(Assert.java:512)
	at org.testng.Assert.assertEqualsImpl(Assert.java:134)
	at org.testng.Assert.assertEquals(Assert.java:115)
	at org.testng.Assert.assertEquals(Assert.java:189)
	at org.testng.Assert.assertEquals(Assert.java:199)
	at com.horoscope.util.Horoscope_baseLib.performingRecharge(Horoscope_baseLib.java:520)
	at test.horoscope.Total_horoscope.openingRechargeApp(Total_horoscope.java:49)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:669)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:877)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1201)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:776)
	at org.testng.TestRunner.run(TestRunner.java:634)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:425)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:420)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:385)
	at org.testng.SuiteRunner.run(SuiteRunner.java:334)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1318)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1243)
	at org.testng.TestNG.runSuites(TestNG.java:1161)
	at org.testng.TestNG.run(TestNG.java:1129)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- openingRechargeApp -->
  <testcase name="LaunchingFromAppDrawer" time="12.84" classname="test.horoscope.Total_horoscope"/>
  <testcase name="myZodiacTab" time="14.225" classname="test.horoscope.Total_horoscope"/>
  <testcase name="DOBsetting" time="30.096" classname="test.horoscope.Total_horoscope"/>
  <testcase name="allZodiacTabSelect" time="0.944" classname="test.horoscope.Total_horoscope"/>
  <testcase name="ariesDataDisplayed" time="2.445" classname="test.horoscope.Total_horoscope"/>
  <testcase name="cancerDataDisplayed" time="2.62" classname="test.horoscope.Total_horoscope"/>
  <testcase name="geminiDataDisplayed" time="2.456" classname="test.horoscope.Total_horoscope"/>
  <testcase name="taurusDataDisplayed" time="2.464" classname="test.horoscope.Total_horoscope"/>
  <testcase name="leoDataDisplayed" time="2.58" classname="test.horoscope.Total_horoscope"/>
  <testcase name="virgoDataDisplayed" time="2.876" classname="test.horoscope.Total_horoscope"/>
  <testcase name="libraDataDisplayed" time="2.509" classname="test.horoscope.Total_horoscope"/>
  <testcase name="scorpioDataDisplayed" time="2.64" classname="test.horoscope.Total_horoscope"/>
  <testcase name="saggitariusDisplayed" time="2.628" classname="test.horoscope.Total_horoscope"/>
  <testcase name="capricornDataDisplayed" time="2.8" classname="test.horoscope.Total_horoscope"/>
  <testcase name="aquariusDataDisplayed" time="2.565" classname="test.horoscope.Total_horoscope"/>
  <testcase name="piscesDataDisplayed" time="2.341" classname="test.horoscope.Total_horoscope"/>
  <testcase name="kundaliMatch" time="6.735" classname="test.horoscope.Total_horoscope">
    <failure type="org.openqa.selenium.TimeoutException" message="Expected condition failed: waiting for element to be clickable: By.className: android.view.ViewGroup (tried for 5 second(s) with 500 MILLISECONDS interval)">
      <![CDATA[org.openqa.selenium.TimeoutException: Expected condition failed: waiting for element to be clickable: By.className: android.view.ViewGroup (tried for 5 second(s) with 500 MILLISECONDS interval)
	at org.openqa.selenium.support.ui.WebDriverWait.timeoutException(WebDriverWait.java:82)
	at org.openqa.selenium.support.ui.FluentWait.until(FluentWait.java:231)
	at test.horoscope.Total_horoscope.kundaliMatch(Total_horoscope.java:197)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:669)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:877)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1201)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:776)
	at org.testng.TestRunner.run(TestRunner.java:634)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:425)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:420)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:385)
	at org.testng.SuiteRunner.run(SuiteRunner.java:334)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1318)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1243)
	at org.testng.TestNG.runSuites(TestNG.java:1161)
	at org.testng.TestNG.run(TestNG.java:1129)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
Caused by: org.openqa.selenium.NoSuchElementException: Cannot locate an element using By.className: android.view.ViewGroup
For documentation on this error, please visit: http://seleniumhq.org/exceptions/no_such_element.html
Build info: version: '3.7.1', revision: '8a0099a', time: '2017-11-06T21:07:36.161Z'
System info: host: 'AJAY', ip: '192.168.0.7', os.name: 'Windows 8.1', os.arch: 'amd64', os.version: '6.3', java.version: '1.8.0_151'
Driver info: driver.version: unknown
	at org.openqa.selenium.support.ui.ExpectedConditions.lambda$findElement$0(ExpectedConditions.java:882)
	at java.util.Optional.orElseThrow(Unknown Source)
	at org.openqa.selenium.support.ui.ExpectedConditions.findElement(ExpectedConditions.java:881)
	at org.openqa.selenium.support.ui.ExpectedConditions.access$000(ExpectedConditions.java:43)
	at org.openqa.selenium.support.ui.ExpectedConditions$7.apply(ExpectedConditions.java:205)
	at org.openqa.selenium.support.ui.ExpectedConditions$7.apply(ExpectedConditions.java:201)
	at org.openqa.selenium.support.ui.ExpectedConditions$22.apply(ExpectedConditions.java:643)
	at org.openqa.selenium.support.ui.ExpectedConditions$22.apply(ExpectedConditions.java:640)
	at org.openqa.selenium.support.ui.FluentWait.until(FluentWait.java:208)
	... 26 more
]]>
    </failure>
  </testcase> <!-- kundaliMatch -->
</testsuite> <!-- Default test -->
